routing/routing_integration_tests/speed_camera_notification.cpp

```diff
@@ -0,0 +1,160 @@
+#include "testing/testing.hpp"
+
+#include "routing/routing_integration_tests/routing_test_tools.hpp"
+
+#include "routing/route.hpp"
+#include "routing/routing_callbacks.hpp"
+#include "routing/routing_helpers.hpp"
+#include "routing/routing_session.hpp"
+#include "routing/routing_tests/tools.hpp"
+
+#include "platform/location.hpp"
+#include "platform/measurement_utils.hpp"
+
+#include "geometry/point2d.hpp"
+
+#include <algorithm>
+#include <memory>
+
+using namespace routing;
+using namespace routing::turns;
+using namespace std;
+
+namespace
+{
+string const kCameraOnTheWay = "Speed camera on the way";
+
+location::GpsInfo MoveTo(ms::LatLon const & coords, double speed = -1)
+{
+  static auto constexpr kGpsAccuracy = 0.01;
+
+  location::GpsInfo info;
+  info.m_horizontalAccuracy = kGpsAccuracy;
+  info.m_verticalAccuracy = kGpsAccuracy;
+  info.m_latitude = coords.lat;
+  info.m_longitude = coords.lon;
+  info.m_speedMpS = speed;
+  return info;
+}
+
+void ChangePosition(RoutingSession & routingSession, ms::LatLon const & coords, double speedKmPH = -1)
```

Comment:
-1.0

Reply:
Pls move `RoutingSession & routingSession` to the end of the parm list.

Reply:
done